ml-conversational-analytic-tool/tests/test_commentAnalysis.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from unittest.mock import MagicMock
+
+import commentAnalysis
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    def setUp(self) -> None:
+        global sentiment_analyzer
+        sentiment_analyzer = commentAnalysis.SentimentIntensityAnalyzer()
+        sentiment_analyzer.polarity_scores = MagicMock(return_value={"compound": 1.0})
+        return super().setUp()
+
+    def test_CommentAnalyzer_init(self):
+        analyzer = commentAnalysis.CommentAnalyzer(['test'])
+        self.assertEqual({'test': 0}, analyzer.word_count,)
+
+    def test_analyzeComment(self):
+        analyzer = commentAnalysis.CommentAnalyzer(['test'])
+        result = analyzer.analyzeComment("This is a test PR")
+        analyzer.vader_sentiment = sentiment_analyzer
+        self.assertEqual({'test': 1, 'Sentiment': 0.0, 'Code Blocks': 0}, result)
```

Comment:
How is value of sentiment 0.0 when `analyzer.vader_sentiment = sentiment_analyzer`?
According to line 11, wouldn't the result be 1.0?

Reply:
Yes. 21 and 20 lines were changed. updating it in new PR!